src/CosmosLogStorage.test.ts

```diff
@@ -0,0 +1,211 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
>*/ [](start = 1, length = 2)

nit: not sure how are we organizing test files in sdk. are they living in the same folder as /src?